### ISSUE TITLE
Issue on admission-hook-configuration.yaml

### ISSUE THREAD

**Issue by elangoc** [kelseyhightower/grafeas-tutorial#5]
Hi,
I am getting below error while executing "kubectl apply -f kubernetes/admission-hook-configuration.yaml" to create the image-signature-webook. Please let me the fix if you have. Thanks.

kubectl apply -f kubernetes/admission-hook-configuration.yaml --validate=false

error: unable to recognize "kubernetes/admission-hook-configuration.yaml": no matches for admissionregistration.k8s.io/, Kind=ExternalAdmissionHookConfiguration

~Elango

**Comment by kelseyhightower** [comment]
This is now fixed on master and has been update to use the new Validating Admission Webhooks: https://kubernetes.io/docs/admin/admission-controllers/#validatingadmissionwebhook-alpha-in-18-beta-in-19

**Comment by dragondscv** [comment]
Hi, is there a document about the spec of ValidatingWebhookConfiguration?
I tried to upgrade to 1.9 and the external admission hook broke.
`error: error validating "/etc/k8s/admission-hook-config.yaml": error validating data: ValidationError(ValidatingWebhookConfiguration): unknown field "externalAdmissionHooks" in io.k8s.api.admissionregistration.v1beta1.ValidatingWebhookConfiguration; if you choose to ignore these errors, turn validation off with --validate=false`

Is there a way to use GenericAdmissionWebhook in 1.9?
I checked the documentation and it's still refering to GenericAdmissionWebhook: GenericAdmissionWebhook
